portfolio/src/main/java/com/google/sps/servlets/FetchServlet.java

```diff
@@ -15,22 +15,41 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
-import java.util.Date;
+import java.util.*;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+
 /** Servlet that responds with the current date. */
 @WebServlet("/fetch")
 public class FetchServlet extends HttpServlet {
 
 
     private static final long serialVersionUID = 1L;
 
+
     @Override
```

Comment:
Not formatted properly.

Reply:
Could you elaborate on how it is supposed to be formatted?

Reply:
nevermind i think i fixed it in recent commits